src/cljs/athens/views/buttons.cljs

```diff
@@ -1,90 +0,0 @@
-   :color            (color :body-text-color)
-   :background-color "transparent"
-   :transition-property "filter, background, color, opacity"
-   :transition-duration "0.075s"
-   :transition-timing-function "ease"
-   ::stylefy/manual [[:&:hover {:background (color :body-text-color :opacity-lower)}]
-                     [:&:active
-                      :&:hover:active
-                      :&.is-active {:color (color :body-text-color)
-                                    :background (color :body-text-color :opacity-lower)}]
-                     [:&:active
-                      :&:hover:active
-                      :&:active.is-active {:background (color :body-text-color :opacity-low)}]
-                     [:&:disabled :&:disabled:active {:color (color :body-text-color :opacity-low)
-                                                      :background (color :body-text-color :opacity-lower)
-                                                      :cursor "default"}]
-                     [:span {:flex "1 0 auto"
-                             :text-align "left"}]
-                     [:kbd {:margin-inline-start "1rem"
-                            :font-size "85%"}]
-                     ["> svg" button-icons-style
-                      [(selectors/& (selectors/not (selectors/last-child))) button-icons-not-last-child-style]
-                      [(selectors/& (selectors/not (selectors/first-child))) button-icons-not-first-child-style]
-                      [(selectors/& ((selectors/first-child (selectors/last-child)))) button-icons-only-child-style]]
-                     [:&.is-primary {:color (color :link-color)
-                                     :background (color :link-color :opacity-lower)}
-                      [:&:hover {:background (color :link-color :opacity-low)}]
-                      [:&:active
-                       :&:hover:active
-                       :&.is-active {:color "white"
-                                     :background (color :link-color)}]
-                      [:&:disabled :&:disabled:active {:color (color :body-text-color :opacity-low)
-                                                       :background (color :body-text-color :opacity-lower)
-                                                       :cursor "default"}]]]})
-
-
-;; Components
-
-(stylefy/class "button" buttons-style)
-
-
-(defn button
-  "Keep button interface as close to vanilla hiccup as possible.
-  Dissoc :style :active and :class because we don't want to merge them in directly.
-  Can pass in a :key prop to make react happy, as a :key or ^{:key}. Just works"
-  ([children] [button {} children])
-  ([{:keys [style active primary class] :as props} children]
-   (let [props- (dissoc props :style :active :primary :class)]
-     [:button (use-style (merge buttons-style style)
-                         (merge props- {:class (vec (flatten [(when active "is-active") (when primary "is-primary") class]))}))
-      children])))
```

Comment:
RIP light-wrapper `button` over `:button` @shanberg

Reply:
We made a better one